node-notes-cli/update.js

```diff
@@ -0,0 +1,31 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+readFile('data.json', 'utf8');
+
+const filePath = 'data.json';
+const editId = process.argv[2];
+const newContent = process.argv[3];
+
+async function updateJsonFile() {
+  try {
+    // Read the JSON file
+    const data = await readFile(filePath, 'utf8');
+    const jsonData = JSON.parse(data);
```

Comment:
You repeat these two lines of code in many places. It would be good to encapsulate them into a function.